File: CS/LocalizerExample/CustomSpreadsheetLocalizer.cs
using DevExpress.XtraSpreadsheet.Localization;
using System;

namespace LocalizerExample
{
    class CustomSpreadsheetLocalizer : XtraSpreadsheetLocalizer
    {
        protected override void PopulateStringTable()
        {
            //base.PopulateStringTable();
            foreach (XtraSpreadsheetStringId s in Enum.GetValues(typeof(XtraSpreadsheetStringId)))
            {
                AddString(s, Enum.GetName(typeof(XtraSpreadsheetStringId), s));
            }
        }
    }
}

File: VB/LocalizerExample/CustomSpreadsheetLocalizer.vb
Imports DevExpress.XtraSpreadsheet.Localization
Imports System

Namespace LocalizerExample
    Friend Class CustomSpreadsheetLocalizer
        Inherits XtraSpreadsheetLocalizer

        Protected Overrides Sub PopulateStringTable()
            'base.PopulateStringTable();
            For Each s As XtraSpreadsheetStringId In System.Enum.GetValues(GetType(XtraSpreadsheetStringId))
                AddString(s, System.Enum.GetName(GetType(XtraSpreadsheetStringId), s))
            Next s
        End Sub
    End Class
End Namespace
